src/index.js

```diff
@@ -197,9 +203,11 @@ export function createComplexityLimitRule(
           }
 
           if (cost > maxCost) {
-            context.reportError(new GraphQLError(
-              formatErrorMessage(cost), [node],
-            ));
+            context.reportError(
+              createError ?
```

Comment:
most users would want to just format the error object provided? attach metadata to it etc.

y not just do `formatError(error: graphqlError)` ?